examples/time_series/goes_hek_m25.py

```diff
@@ -38,9 +38,9 @@
 # Finally lets plot everything together
 
 goes.peek()
-plt.axvline(parse_time(flares_hek[0].get('event_peaktime')))
-plt.axvspan(parse_time(flares_hek[0].get('event_starttime')),
-            parse_time(flares_hek[0].get('event_endtime')),
+plt.axvline(parse_time(flares_hek[0].get('event_peaktime')).datetime)
```

Comment:
This is unfortunate. I wonder if we could implement a matplotlib converter to automatically support Time. @dstansby does that sound like a thing that could be done?

Reply:
Hahaha, welcome to trying to use units with axvline! Is there a Time units converter for normal plotting already in astropy? If so this should work on certain minimum versions of astropy/matplotlib since I put some fixes in both recently.

Reply:
I think we can use `Time.plot_date` here instead of `.datetime`. As mentioned in [this comment](https://github.com/astropy/astropy/issues/408#issuecomment-14653225), it should be more efficient.

Reply:
nice